massmigration/templates/massmigration/manage_migrations.html

```diff
@@ -8,23 +8,40 @@ <h1>Migrations</h1>
 		<tr>
 			<th>App</th>
 			<th>Migration</th>
```

Comment:
This is to create this. It's mostly for when you've only got one DB and so having a column of "default" might be confusing because you wouldn't know what it means. But also I think it makes it a bit nicer in general.
<img width="957" alt="Screenshot 2024-03-06 at 13 11 54" src="https://github.com/flea89/django-mass-migration/assets/270255/2f48c756-a430-4900-bae9-1ac29dc5057d">